vectordraw.js

```diff
@@ -10,7 +10,8 @@ var VectorDraw = function(element_id, settings) {
         vectors: [],
         points: [],
         expected_result: {},
-        show_navigation: false
+        show_navigation: false,
+        add_vector_label: "Add Selected Force"
```

Comment:
@dseaton this line specifies default value - as you can see it's the same as we used previously.